packages/legend-studio/src/components/editor/side-bar/Explorer.tsx

```diff
@@ -190,14 +191,14 @@ const ExplorerContextMenu = observer(
     return (
       <MenuContent data-testid={CORE_TEST_ID.EXPLORER_CONTEXT_MENU}>
         {extraExplorerContextMenuItems}
-        <MenuContentItem>Rename (WIP)</MenuContentItem>
-        {node && (
-          <MenuContentItem onClick={deleteElement}>Delete</MenuContentItem>
-        )}
-        {node && (
-          <MenuContentItem onClick={openElementInViewerMode}>
-            View in Project
-          </MenuContentItem>
+        {isNotInViewerMode && <MenuContentItem>Rename (WIP)</MenuContentItem>}
+        {isNotInViewerMode && node && (
+          <>
+            <MenuContentItem onClick={deleteElement}>Delete</MenuContentItem>
+            <MenuContentItem onClick={openElementInViewerMode}>
```

Comment:
can we enable 'copy link element' in viewer mode. users would still want to share the element link.

Reply:
`Copy link` works in viewer mode, I can confirm.